martin/martin-ui/src/components/catalogs/font.tsx

```diff
@@ -0,0 +1,168 @@
+          <p className="text-muted-foreground">Preview all available font assets</p>
+        </div>
+        <div className="relative w-full md:w-64">
+          <Search className="absolute left-3 top-1/2 transform -translate-y-1/2 text-gray-400 w-4 h-4" />
+          <Input
+            className="pl-10 md:w-64 w-full bg-card"
+            onChange={(e) => onSearchChangeAction(e.target.value)}
+            placeholder="Search fonts..."
+            value={searchQuery}
+          />
+        </div>
+      </div>
+
+      <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-6">
+        {filteredFonts.map(([name, font]) => (
+          <Card className="hover:shadow-lg transition-shadow" key={name}>
+            <CardHeader>
+              <div className="flex items-center justify-between">
+                <div className="flex items-center space-x-2">
+                  <Type className="w-5 h-5 text-primary" />
+                  <CardTitle className="text-lg font-mono">{name}</CardTitle>
+                </div>
+                {font.format && (
+                  <Badge className="uppercase" variant="secondary">
+                    {font.format}
+                  </Badge>
+                )}
+              </div>
+              <CardDescription>
+                Family: {font.family} • Style: {font.style}
+              </CardDescription>
+            </CardHeader>
+            <CardContent>
+              <div className="space-y-4">
+                <div className="p-3 bg-gray-50 text-gray-900 rounded-lg">
+                  <p className="text-sm font-medium mb-2 text-gray-900">Preview:</p>
+                  <Tooltip>
+                    <TooltipTrigger className="cursor-help">
+                      <p
+                        className="text-base text-gray-900 blur-sm animate-pulse"
+                        style={{ fontFamily: font.family, fontWeight: 500 }}
+                      >
+                        The quick brown fox jumps over the lazy dog
+                      </p>
+                    </TooltipTrigger>
+                    <TooltipContent>
+                      <p>Not currently implemented in the frontend</p>
+                    </TooltipContent>
+                  </Tooltip>
+                </div>
+                <div className="space-y-2 text-sm text-muted-foreground">
+                  <div className="flex justify-between">
+                    <span>Glyph count:</span>
+                    <span>{font.glyphs}</span>
+                  </div>
+                </div>
+                <div className="flex flex-col md:flex-row items-center gap-2 mt-4">
+                  <CopyLinkButton
+                    className="flex-1 bg-transparent w-full"
+                    link={buildMartinUrl(`/font/${name}/{range}`)}
+                    size="sm"
+                    toastMessage="Font link copied!"
+                    variant="outline"
+                  />
```

Comment:
This is clearly not what users want.

not quite sure what users want instead 🤷🏻‍♂️
I doubt that they would want to get this URL.

<img width="1310" height="735" alt="image" src="https://github.com/user-attachments/assets/8d13a60b-cb5b-48c3-8bcf-1d8a55f0796c" />